src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -169,7 +168,10 @@ public enum CaseEvent {
     NOTIFY_CASEWORKER_DJ_RECEIVED(CAMUNDA),
     ADD_PDF_TO_MAIN_CASE(CAMUNDA),
     JUDICIAL_REFERRAL(CAMUNDA),
-    SERVICE_REQUEST_RECEIVED(CAMUNDA);
+    SERVICE_REQUEST_RECEIVED(CAMUNDA),
+    migrateCase(CAMUNDA),
```

Comment:
Is this move of migrateCase intended, or a conflict resolution mistake?

Reply:
I do seem to notice that it also change from "USER" to "CAMUNDA" so I believe it's intentional.